fs_mgr/fs_mgr_avb.cpp

```diff
@@ -190,10 +190,14 @@ class FsManagerAvbVerifier {
 
 std::unique_ptr<FsManagerAvbVerifier> FsManagerAvbVerifier::Create() {
     std::string cmdline;
+#if !defined(ANDROID_CONTAINER)
     if (!android::base::ReadFileToString("/proc/cmdline", &cmdline)) {
         LERROR << "Failed to read /proc/cmdline";
         return nullptr;
     }
+#else
+    cmdline = getenv("INIT_KERNEL_CMDLINE");
+#endif
```

Comment:
AVB should be in the first stage only. In fact, probably not important for containers at all.